packages/graphback-runtime-knex/src/PgKnexDBDataProvider.ts

```diff
@@ -11,7 +11,8 @@ import { KnexDBDataProvider } from './KnexDBDataProvider';
  * that works with the rest of the databases.
  */
 //tslint:disable-next-line: no-any
-export class PgKnexDBDataProvider<Type = any, GraphbackContext = any> extends KnexDBDataProvider<Type, GraphbackContext>{
+
+export class PgKnexDBDataProvider<Type = any, GraphbackContext = any> extends PostgreDBDataProvider<Type, GraphbackContext>{
```

Comment:
Let's not change any of this code. We should create only `PostgreDBDataProvider` as new file and provide it as an alternative to existing implementations

Reply:
ok... I'll create a new PR on this